### ISSUE TITLE
Next / previous Track in NET/USB not working for me

### ISSUE THREAD

**Issue by quelbs** [mkulesh/onpc#44]
The keys do not work when listening to mp3

```
D/ViewRootImpl@acddcbc[MainActivity]: ViewPostImeInputStage processPointer 0
D/ViewRootImpl@acddcbc[MainActivity]: ViewPostImeInputStage processPointer 1
D/onpc: StateManager: sending message: NTC[null; ZONE_INDEX=0; CMD=TRUP]
D/onpc: MessageChannel: >> sending: ISCP/v1[16,10]: NTC(TRUP)
D/ViewRootImpl@acddcbc[MainActivity]: ViewPostImeInputStage processPointer 0
D/ViewRootImpl@acddcbc[MainActivity]: ViewPostImeInputStage processPointer 1
D/onpc: StateManager: sending message: NTC[null; ZONE_INDEX=0; CMD=TRDN]
D/onpc: MessageChannel: >> sending: ISCP/v1[16,10]: NTC(TRDN)
```

For TX-8150 ´NTC(RIGHT)´ and ´NTC(LEFT)´ would work fine.

**Comment by mkulesh** [comment]
Hi @quelbs, it is really unexpected (:
Could you please test whether ALL network services have the same behavior; i.e NTC(RIGHT) and NTC(LEFT are services-independent. The TX-8150 is unfortunately not listed in the [device matrix](https://github.com/mkulesh/onpc/blob/master/doc/ISCP_AVR_140.xlsx), but, for example, TX-8270 supports  TRUP and TRDN. If this behavior is service-independent, we can just fix the method [onCreateView](https://github.com/mkulesh/onpc/blob/master/app/src/main/java/com/mkulesh/onpc/MonitorFragment.java) and create btnPrevious and btnNext device-dependent.

**Comment by mkulesh** [comment]
on some receivers, "TRDN" and "TRUP" for USB only work in playback mode. Proposed solution: switch to this mode before send OperationCommandMsg if current mode is LIST

**Comment by quelbs** [comment]
Your solution works fine with my receiver. Thank you for the adjustments.

**Comment by mkulesh** [comment]
Released in version 1.0
